pilot/pkg/serviceregistry/mcp/controller.go

```diff
@@ -312,6 +312,11 @@ func (c *controller) Update(config.Config) (newRevision string, err error) {
 	return "", errUnsupported
 }
 
+func (c *controller) UpdateStatus(config.Config) (string, error) {
+	log.Warnf("updateStatus: %s", errUnsupported)
+	return "", errUnsupported
```

Comment:
Why is this error different from the "not implemented" error in the other classes?

Reply:
not sure. just mirrored what the other controller methods did.

Reply:
It's a little different. We have configs stores that are "writers" which are only memory and k8s. Everything else is read-only by design IIUC.